bookwyrm/templates/author/author.html

```diff
@@ -22,42 +22,59 @@ <h1 class="title">{{ author.name }}</h1>
     </div>
 </div>
 
-<div class="block content columns">
-    {% if author.aliases or author.born or author.died or author.wikipedia_link %}
+<div class="block content columns" itemscope itemtype="https://schema.org/Person">
+    <meta itemprop="name" content="{{ author.name }}">
+
+    {% if author.aliases or author.born or author.died or author.wikipedia_link or author.openlibrary_key or author.inventaire_id %}
     <div class="column is-narrow">
         <div class="box">
             <dl>
                 {% if author.aliases %}
                 <div class="is-flex">
                     <dt class="mr-1">{% trans "Aliases:" %}</dt>
-                    <dd itemprop="aliases">{{ author.aliases|join:', ' }}</dd>
+                    <dd>
+                        {% for alias in author.aliases %}
+                        <span itemprop="alternateName">{{alias}}</span>{% if not forloop.last %},{% endif %}
+                        {% endfor %}
+                    </dd>
```

Comment:
I understand you used the `<span>` to avoid including the `,` in the value, which is great. I think the markup could be simplified to the following and would be slightly more semantic regarding the description list. What do you think?
```html
                    {% for alias in author.aliases %}
                        <dd itemprop="alternateName" content="{{ alias }}">
                            {{ alias }}{% if not forloop.last %},{% endif %}
                        </dd>
                    {% endfor %}
```

Reply:
I forgot about multiple `dd`!